L1Trigger/TrackFindingTracklet/test/HybridTracksNewKF_cfg.py

```diff
@@ -0,0 +1,76 @@
+################################################################################################
+# To run execute do
+# cmsRun L1Trigger/TrackFindingTracklet/test/HybridTracksNewKF_cfg.py
+# where the arguments take default values if you don't specify them. You can change defaults below.
+#################################################################################################
+
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process( "Demo" )
+process.load( 'Configuration.Geometry.GeometryExtended2026D76Reco_cff' ) 
+process.load( 'Configuration.Geometry.GeometryExtended2026D76_cff' )
+process.load( 'Configuration.StandardSequences.MagneticField_cff' )
+process.load( 'Configuration.StandardSequences.FrontierConditions_GlobalTag_cff' )
+process.load( 'Configuration.StandardSequences.L1TrackTrigger_cff' )
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgradePLS3', '')
+
+# load code that associates stubs with mctruth
+process.load( 'SimTracker.TrackTriggerAssociation.StubAssociator_cff' )
+# load code that produces DTCStubs
+process.load( 'L1Trigger.TrackerDTC.ProducerED_cff' )
+# load code that analyzes DTCStubs
+process.load( 'L1Trigger.TrackerDTC.Analyzer_cff' )
+# L1 tracking => hybrid emulation 
+process.load("L1Trigger.TrackFindingTracklet.L1HybridEmulationTracks_cff")
+#--- Load code that analyzes hybrid emulation 
+process.load( 'L1Trigger.TrackFindingTracklet.Analyzer_cff' )
+# load code that fits hybrid tracks
+process.load( 'L1Trigger.TrackFindingTracklet.ProducerKF_cff' )
+
+# build schedule
+process.mc = cms.Sequence( process.StubAssociator )
+process.dtc = cms.Sequence( process.TrackerDTCProducer + process.TrackerDTCAnalyzer )
+process.tracklet = cms.Sequence( process.L1TrackletTracks + process.TrackFindingTrackletAnalyzerTracklet )
+process.interIn = cms.Sequence( process.TrackFindingTrackletProducerKFin + process.TrackFindingTrackletAnalyzerKFin )
+process.kf = cms.Sequence( process.TrackFindingTrackletProducerKF + process.TrackFindingTrackletAnalyzerKF )
+process.TTTracks = cms.Sequence( process.TrackFindingTrackletProducerTT + process.TrackFindingTrackletProducerAS )
+process.interOut = cms.Sequence( process.TrackFindingTrackletProducerKFout + process.TrackFindingTrackletAnalyzerKFout )
+process.tt = cms.Path( process.mc + process.dtc + process.tracklet + process.interIn + process.kf + process.TTTracks + process.interOut )
+process.schedule = cms.Schedule( process.tt )
+
+# create options
+import FWCore.ParameterSet.VarParsing as VarParsing
+options = VarParsing.VarParsing( 'analysis' )
+# specify input MC
```

Comment:
To avoid needing to list lots of MC samples here, copying style of L1TrackNtupleMaker_cfg.py, replace this by:

# specify input MC

#--- To use MCsamples scripts, defining functions get*data*() for easy MC access,
#--- follow instructions in https://github.com/cms-L1TK/MCsamples
#from MCsamples.Scripts.getCMSdata_cfi import *
#from MCsamples.Scripts.getCMSlocaldata_cfi import *

# Read data from card files (defines getCMSdataFromCards()):
#from MCsamples.RelVal_1130_D76.PU200_TTbar_14TeV_cfi import *
#inputMC = getCMSdataFromCards()

# Or read .root files from directory on local computer:
#dirName = "$myDir/whatever/"
#inputMC=getCMSlocaldata(dirName)

# Or read specified dataset (accesses CMS DB, so use this method only occasionally):
#dataName="/RelValTTbar_14TeV/CMSSW_11_3_0_pre6-PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/GEN-SIM-DIGI-RAW"
#inputMC=getCMSdata(dataName)

# Or read specified .root file:
inputMC = [
'/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/00026541-6200-4eed-b6f8-d3a1fd720e9c.root'
]
options.register( 'inputMC', inputMC, VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.string, "Files to be processed" )

Reply:
The last time I put something into an official release the reviewers where not happy with me storing root files in a separate file and forced my to do it who it is done right now. Did you double check if using your MC samples script method will pass official cmssw code review?

Reply:
It does also not feel appealing that I have to additionally checkout something from somewhere. Is there a chance that these scripts will make it into a release?

Reply:
This python fragment is already in the official CMSSW release in L1TrackNtupleMaker_cfg.py. The MCsamples code is commented out so that the job runs without needing to check anything out, but that it is as easy as possible to use the MCsamples tools. We can't commit the MCsamples tools to central CMSSW, as they contain lists of MC datasets, which are not allowed there.

Reply:
done